README.md

```diff
@@ -60,6 +60,44 @@ When make completes, three services will be running on localhost:
 
 Go to the complete [installation instructions](https://mailchimp.com/developer/open-commerce/guides/quick-start/#access-the-dashboard-playground-and-storefront) to see how to set up your store
 
+# API development
+
+## What you need
+* NPM >= 7 (support workspaces)
+* Docker & docker-compose
+* MongoDB
```

Comment:
This is incorrect, you don't need Mongo installed if you are using the docker-compose and most people are better of not installing it locally so you can have different instances of it